src/cli.ts

```diff
@@ -248,7 +269,8 @@ async function main() {
   } else if (argv._ && argv._[0] === 'validate') {
     await validate({
       changelogPath,
-      currentVersion,
+      // TODO: Is it OK to assign an empty string here?
+      currentVersion: currentVersion || '',
```

Comment:
This seems bad. Should we add some input validation here?

Reply:
See this comment in `updateChangelog`:
>@param options.currentVersion - The current version. Required if `isReleaseCandidate` is set, but optional otherwise.

I believe it was my intention to consider it "conditionally required" in the same way for `validate` as well, but I forgot to explain this in the doc comment.

Reply:
This would mean making this optional in `ValidateOptions` as well

Reply:
Done in: 203c240